packages/components/src/ErrorInformation/index.js

```diff
@@ -0,0 +1,3 @@
+// @flow
+
+export { default as ErrorInformation } from './ErrorInformation';
```

Comment:
This file is not necessary, after #98 discussion we settled on structure without extra `index.js` files.

Reply:
OK, that's better :)